samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -509,6 +510,28 @@ static void button_handler(uint32_t button_states, uint32_t has_changed)
 }
 #endif
 
+#if defined(CONFIG_SHELL)
+static int shell_publish(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	int ret;
+
+	ret = data_publish(&client,
+				MQTT_QOS_1_AT_LEAST_ONCE,
+				CONFIG_BUTTON_EVENT_PUBLISH_MSG,
+				sizeof(CONFIG_BUTTON_EVENT_PUBLISH_MSG)-1);
```

Comment:
Alignment seems off here.

Reply:
seems what my IDE sees is different from what is shown here.

Reply:
Perhaps you have 4 spaces/tab configured locally? It still seems off:
![image](https://user-images.githubusercontent.com/22167799/193026051-3db1033c-17ab-419b-b683-e4ee0edeba11.png)

Reply:
Please check the update